lib/aca_entities/magi_medicaid/mitc/applicant.rb

```diff
@@ -8,7 +8,7 @@ class Applicant < Dry::Struct
         # @!attribute [r] person_id
         # An integer representing the applicant.
         # @return [Integer]
-        attribute :person_id,        Types::Integer.meta(omittable: false)
+        attribute :person_id,        Types::String.meta(omittable: false)
```

Comment:
@4till2 @mdkaraman  MitC expects us to send **integer** for **person_id** and this is why the contract validations are implemented this way. You might want to change the way we generate person hbx_id in Enroll which will fix the issue of submitting financial assistance applications.

Reply:
Ok thanks for the clarity. We will move forward in that direction